### PR TITLE
Proofread README.md

### PR DESCRIPTION
**Pull request by haujason**
Syntax Errors
-Added 'some'
-Added 'the'

-'in' -> 'for the'

Add -ing only when the verb should be progressive/continuous
-'calling' -> 'call'
-'iterating' -> 'iterate'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Planck.js is [available on jsDelivr](https://www.jsdelivr.com/projects/planck).
 ## Usage
 
 Planck.js does not use any renderer by default. To use or integrate it with a rendering library all you need to do 
-is calling `world.step(1/hz)` in each frame, and then iterating over world entities to draw or update them.
+is call `world.step(1/hz)` in each frame, and then iterate over world entities to draw or update them.
```
Comment (shakiba) [inline README.md]: @haujason Are you sure about this? Because here `calling` is used as a none, not verb.

File: README.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 # Planck.js<sup>&alpha;</sup>
 
-Planck.js is JavaScript rewrite of Box2D physics engine for cross-platform HTML5 game development. **[Check out demos!](http://piqnt.com/planck.js/)**
+Planck.js is JavaScript rewrite of Box2D physics engine for cross-platform HTML5 game development. **[Check out some demos!](http://piqnt.com/planck.js/)**
```
Comment (shakiba) [inline README.md]: @haujason I prefer to keep this line short!

File: README.md (hunk 1 above)
Comment (haujason) [inline README.md]: I assume you mean noun? In that case then calling is correct, though I admit then I struggle to understand its usage here.

File: README.md (hunk 2 above)
Comment (haujason) [inline README.md]: It's a matter of formality and style, if that is what you want then you can keep it that way

**Pull request merged**